src/axom/bump/tests/blueprint_testing_helpers.hpp

```diff
@@ -530,7 +530,7 @@ bool compare_views(const Container1 &a, const Container2 &b)
   }
   if(!eq)
   {
-    axom::fmt::format("a={{{}}}\nb={{{}}}", axom::fmt::join(a, ","), axom::fmt::join(b, ","));
+    (void)axom::fmt::format("a={{{}}}\nb={{{}}}", axom::fmt::join(a, ","), axom::fmt::join(b, ","));
```

Comment:
I assume this silences the warning, but I'm not sure that it's the right resolution.
I think the issue here is that we're doing work with `fmt::format()` but not doing anything with the result.

@BradWhitlock -- did you intend to print/output/return the result?

Reply:
I would think that any reasonable compiler would not complain if the return value of a method is ignored by the caller unless it is marked [[nodiscard]], for example.

If we cast to void, I think it would be better to use C++ style: static_cast<void> rather than C style (void)

Reply:
It is marked as nodiscard -- https://github.com/fmtlib/fmt/pull/2612 -- and we're seeing it now b/c we're compiling in `c++17`

My question was about why we're formatting these strings and not outputting/returning them.
It seems like an oversight on our end.